client/src/visualization/mapContainer.tsx

```diff
@@ -18,7 +17,12 @@ import Deserializer, {
   mqttMessageToGeo,
   MqttMessage,
 } from '../location/mqttDeserialize';
+import { useUbiMqtt } from '../location/mqttConnection';
 import BluetoothNameModal from './bluetoothNameModal';
+import raspberryLogo from '../../asset/rasp.png';
+import { RaspberryLocation } from './calibrationPanel';
+import { StaticUbiMarker, OfflineMarker, NonUserMarker } from './marker';
+import { Location } from '../common/typeUtil';
 
 const KUMPULA_COORDS = { lat: 60.2046657, lon: 24.9621132 };
 const DEFAULT_NONTRACKED_ZOOM = 12;
```

Comment:
This has nothing in particular to do with this PR.

Wouldn't it be better if this default zoom value was larger, for instance 17.

Reply:
Good idea